### PR TITLE
update usage with tfe.local_computation decorator

### PR DESCRIPTION
**Pull request by jvmncs**
Merge after tf-encrypted/tf-encrypted#598

### CODE REVIEW

File: pages/index.js
Comment (bendecoste) [inline pages/index.js]: input shape is not defined anymore

**Review by bendecoste — DISMISSED** [review]

**Review by bendecoste — APPROVED** [review]

**Comment by bendecoste** [comment]
> Merge after [tf-encrypted/tf-encrypted#598](https://github.com/tf-encrypted/tf-encrypted/pull/598)

This brings up an interesting challenge.  Really this probably shouldn't merge until this PR is both merged and released.

**Comment by jvmancuso** [comment]
agreed -- it's been merged but not released, so let's merge this when it's been released this week

**Pull request merged**
